tests/wallet/test_wallet.py

```diff
@@ -777,7 +777,7 @@ async def test_clawback_resync(
         await server_2.start_client(PeerInfo(self_hostname, uint16(server_1._port)), None)
         await server_3.start_client(PeerInfo(self_hostname, uint16(server_1._port)), None)
         expected_confirmed_balance = await full_node_api.farm_blocks_to_wallet(count=num_blocks, wallet=wallet)
```

Comment:
does this test fail without this resync fix?

Reply:
No, but it doesn't cover the issue. That's why we didn't find this bug.

Reply:
lets add a test that covers this issue

Reply:
This change will cover the case